src/main/webapp/js/finscholar/scholarshippageview/scholarshippageview.soy

```diff
@@ -0,0 +1,109 @@
+{namespace example.templates.scholarshippageviews}
+{template .scholarshippage}
+  {@param scholarship: [generalInfo: [scholarshipName: string,  
+                                       schoolsList:string,
+                                       introduction: string,
+                                       URL: uri],
+                        requirements: [academicRequirements: string,
+                                       ethnicityRaceRequirements: string,
+                                       financialRequirements: string,
+                                       genderRequirements: string,
+                                       locationRequirements: string,
+                                       nationalOriginRequirements: string,
+                                       otherRequirements: string],
+                        applicationNotes: [isRenewable: bool,
+                                           amountPerYear: string,
+                                           applicationProcess: string,
+                                           numberOfYears: int]
+                        ]}
+  <section class="section--center mdl-grid mdl-grid--no-spacing mdl-shadow--2dp">
+    <div class="mdl-card__supporting-text mdl-grid mdl-grid--no-spacing">
+      <div class="page-content">
+        {call .scholarshipIntroduction}
+          {param generalInfo: $scholarship.generalInfo /}
+        {/call}
+        {call .scholarshipRequirements}
+          {param requirements: $scholarship.requirements /}
+        {/call}
+        {call .scholarshipApplication}
+          {param applicationNotes: $scholarship.applicationNotes /}
+        {/call}
+      </div>
+    </div>
+  </section>
+{/template}
+
+{template .scholarshipIntroduction}
+  {@param generalInfo: [scholarshipName: string,  
+                         schoolsList: string,
+                         introduction: string,
+                         URL: uri]}
+  <div class="mdl-card mdl-cell mdl-cell--12-col">
+    <div id="scholarship-header">
+      <h4>{$generalInfo.scholarshipName}</h4>
+      <a href={$generalInfo.URL}>Visit official website</a>
+      <p> {$generalInfo.schoolsList} </p>
+    </div>
+
+  </div>
+  <div id="introduction">
+    <p>Introduction: {$generalInfo.introduction}</p>
+  </div>
+{/template}
+
+{template .scholarshipRequirements}
+  {@param requirements: [academicRequirements: string, 
+                         financialRequirements: string,
+                         ethnicityRaceRequirements: string,
+                         genderRequirements: string,
+                         locationRequirements: string,
+                         nationalOriginRequirements: string,
+                         otherRequirements: string]}
+  <div class="mdl-card mdl-cell mdl-cell--10-col">
+    <h4 class="mdl-cell mdl-cell--12-col">Requirements</h4>
+    <div class="requirement section__text mdl-cell mdl-cell--10-col-desktop">
+      <h6> Academic Requirements </h6>
```

Comment:
Maybe using (`<h6>`) is not such a good idea as Carlos mentioned with issues with a screen reader.